migrations/2021-06-21-093252_create_games/up.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE games (
+    id INTEGER PRIMARY KEY
+);
+
+-- Going to create a separate table for players to avoid player_1_id, player_2_id etc columns in future.
+CREATE TABLE players (
+    id INTEGER PRIMARY KEY,
+    game_id INTEGER NOT NULL,
+    user_id INTEGER NOT NULL,
+    won BOOLEAN, -- Can be set per player, so that in future, multiple players (e.g. in a team) can win.
+    elo_change INTEGER,
+    FOREIGN KEY (game_id) REFERENCES games (id),
+    FOREIGN KEY (user_id) REFERENCES users (id),
+    UNIQUE (game_id, user_id)
+);
```

Comment:
I imagine this should probably also need a game identifier -- like, which specific game was this played on? was it played in last week's round 1 game, or are we in round 5 at this point being an entirely different game?

Reply:
Intention was to add a tournament_id column in future.

Reply:
I think we should add it now -- it's definitely something we will need, so means there's 1 less migration in future